kolibri/plugins/learn/assets/src/views/exam-list/index.vue

```diff
@@ -70,6 +70,7 @@
           params: {
             channel_id: exam.channelId,
             id: exam.id,
+            questionNumber: exam.answerCount,
```

Comment:
For an unstarted exam, `answerCount` is underfined @jonboiser

Reply:
Should this be defaulted to 0 then? Or should it always be 0, since `answerCount` might not be the last question attempted (e.g. user skipped around).

Reply:
And why is it undefined? If the exam wasn't started, isn't `answerCount = 0` also correct?

Reply:
Note, this is why there is an `EXAM_ROOT` that does not need this questionNumber parameter. Admittedly, just setting the questionNumber here to 0 would also achieve the same.

We should always enter the exam on the zeroth question.